test/anno_verify_assignment_test1/main.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include "../../Annotations/Annotations.h"
+
+struct A {
+    char* a MustCall("foo1");
+};
+
+struct B {
+    char* a MustCall("foo1, foo2");
+};
+
+char* MustCall("foo1") set_mc1(char* a) {
+    return a;
+}
+
+char* MustCall("foo2") set_mc2(char* a) {
+    return a;
+}
+
+int main() {
+    struct A a;
+    struct B b;
+
+    a.a = set_mc1(a.a); // Warning -- missing MustCall("foo1") on param 0 of set_mc1
```

Comment:
I thought that we'd decided on Tuesday that this warning isn't necessary, because in this case we know that a.a has not been initialized (and can therefore be treated as `MustCall({})`). To be clear, I don't object to warning here: it's sound but not precise to do so.

Reply:
I'll update to comment to state it should not have a warning there